proxy/README.md

```diff
@@ -0,0 +1,99 @@
+# `cc-proxy`
+
+`cc-proxy` is a daemon offering access to the VM agent to multiple clients on
+the host. It exposes an API accessible through an `AF_UNIX` socket.
+
+This socket can be found at: `${localstatesdir}/run/cc-oci-runtime/proxy.sock`.
+
+## Protocol
```

Comment:
You may want to extensively comment the prototype implementation and turn the generated godoc into a spec of some sort. But at some point this protocol needs to be formalized a bit further.